src/cmd-buildextend-metal

```diff
@@ -208,23 +208,15 @@ echo "Estimating disk size..."
 # and doing so has apparently negative performance implications.
 /usr/lib/coreos-assembler/estimate-commit-disk-size ${BLKSIZE:+--blksize ${BLKSIZE}} --repo "$ostree_repo" "$commit" --add-percent 35 > "$PWD/tmp/ostree-size.json"
 rootfs_size="$(jq '."estimate-mb".final' "$PWD/tmp/ostree-size.json")"
-# extra size is the non-ostree partitions, see create_disk.sh
-image_size="$(( rootfs_size + 513 ))M"
-echo "Disk size estimated to ${image_size}"
-
-# For bare metal and dasd images, we use the estimated image size. For IaaS/virt, we get it from
-# image.yaml because we want a "default" disk size that has some free space.
-case "${image_type}" in
-    metal*|dasd)
-        # Unset the root size, which will inherit from the image size
-        rootfs_size=0
-        ;;
-    qemu)
-        image_size="$(jq -r ".size" < "${image_json}")G"
-        rootfs_size="${rootfs_size}M"
-        ;;
-    *) fatal "unreachable image_type ${image_type}";;
-esac
+# The minimum size of a disk image we'll need will be the rootfs_size
+# estimate plus the size of the non-root partitions. We'll use this
+# size for the metal/dasd images, but for the IaaS/virt image we'll use
+# the size set in the configs since some of them have minimum sizes that
+# the platforms require and we ant a "default" disk size that has some
```

Comment:
```suggestion
# the platforms require and we want a "default" disk size that has some
```

Reply:
I'll make this change in the next updload.

Reply:
fixed